plugin/pkg/scheduler/factory/factory.go

```diff
@@ -1109,6 +1109,11 @@ func (factory *configFactory) MakeDefaultErrorFunc(backoff *util.PodBackoff, pod
 		} else {
 			if _, ok := err.(*core.FitError); ok {
 				glog.V(4).Infof("Unable to schedule %v %v: no fit: %v; waiting", pod.Namespace, pod.Name, err)
+			} else if errors.IsNotFound(err) {
+				if errStatus, ok := err.(errors.APIStatus); ok && errStatus.Status().Details.Kind == "node" {
+					node := v1.Node{ObjectMeta: metav1.ObjectMeta{Name: errStatus.Status().Details.Name}}
```

Comment:
Can we try again to get the node and if we still the "not found" error, then remove the node?

Reply:
@bsalamat Yeah, that is good, I will fix it.

Reply:
@bsalamat Done, PTAL